File: WPA.backend/WPA.backend/DTOs/Guests/CreateGuestDto.cs
using System.ComponentModel.DataAnnotations;
using WPA.backend.Entities;

namespace WPA.backend.DTOs.Guests
{
    public class CreateGuestDto
    {
        public int PlannerId { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public bool IsTravelling { get; set; }

        public string Adnotation { get; set; }

        [Required]
        public GuestStatus Status { get; set; }

        [Required]
        public GuestSide Side { get; set; }
    }
}

File: frontend/src/app/core/api/models/create-guest-dto.ts
/* tslint:disable */
import { GuestSide } from './guest-side';
import { GuestStatus } from './guest-status';
export interface CreateGuestDto {
  adnotation?: string;
  isTravelling: boolean;
  name: string;
  plannerId: number;
  side: GuestSide;
  status: GuestStatus;
}
